tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -95,6 +95,7 @@ func newStressRangePrefix(kvc pb.KVClient, keySuffixRange int) stressFunc {
 		_, err := kvc.Range(ctx, &pb.RangeRequest{
 			Key:      []byte("foo"),
 			RangeEnd: []byte(fmt.Sprintf("foo%d", rand.Intn(keySuffixRange))),
```

Comment:
I don't think this needs to be randomized any more since there's a limit? It can just be fmt.Sprintf("foo%d", keySuffixRange) plus a random limit